src/utils/index.ts

```diff
@@ -0,0 +1,32 @@
+import { dataProvider } from '../providers/dataProvider';
+
+export enum AuditType {
+	LOGIN = 'login',
+	LOGOUT = 'logout',
+	CREATE_USER = 'create_user',
+	DELETE_USER = 'delete_user',
+	SHOW_USER = 'show_user',
+	EDIT_USER = 'edit_user',
+	LIST_USERS = 'list_users',
+	LIST_AUDIT = 'list_audit',
+}
+
+export const trackEvent = (
+	type: AuditType,
+	activity_detail?: string,
+	defaultUser?: User
+) => {
+	const userData = localStorage.getItem('token');
```

Comment:
@salony75way - ideally this code would not be aware that we are using `localStorage`.  It would be more versatile if it was dealing with the `dataProvider`.

So, when we move away from `localStorage` to a real REST backend, the code would still work.

Reply:
Or, if this code is getting the token from localStorage - I would rather it was wrapped in another utility method. `trackEvent` should not have to know how we store user tokens.

Reply:
Done.